code/Bresenham Line Algorithm Programs/Bresenham Line Algo.py

```diff
@@ -0,0 +1,65 @@
+# Import graphics library
```

Comment:
Please autoformat the code using black. Move the file to the proper directory as well.

Reply:
Hi, @arnavb can you please elaborate what i need to do? I mean what do you mean by '' autoformat the code using black''

Reply:
@robotjellyzone [Black](https://black.readthedocs.io/en/stable/) is a python code formatter used to make the code look clean.